utils/idMapping.js

```diff
@@ -0,0 +1,37 @@
+const { fetchUser } = require('../models/users')
```

Comment:
Can we change the file name as users.js, so that in future, we can add other utilities related to user data?

Reply:
yeah ok